### DISCUSSION TITLE
Amber Electric (Australia) prices, choose the source

### DISCUSSION THREAD

**Discussion by rob0101** [netzero-labs/netzero#32] [Ideas]
Hi again.

would it be possible to offer a toggle between the Amber forecasts and the AEMO forecasts ?

Amber's predictions are mostly fine except they can be missing some critical information that makes them less reliable at times.

An example:  today in South Australia we had a 20 minute price spike which AEMO was flagging 24+ hours ahead, but Amber was overruling.  Amber looked at the demand, weather, recent pricing and called the AEMO price BS.    The problem was that AEMO (and those of us watching closer) could see that a vital piece of infrastructure was going to be unavailable, leaving the state vulnerable to needing more expensive alternative fuels.

These events happen infrequently, but they are times when costs/income are at their extremes.

This morning, TBC could not see the spike coming and had no need to prepare (top up) the battery in advance.  There are quite a few people who would want the pessimistic AEMO prices on these days rather than the optimistic Amber prices.

Thanks again for all your work.

**Comment by zigam** [comment]
Thanks for the suggestion! The challenge with a feature like this is making it understandable for users that are not as familiar with the intricacies of electricity markets.

**Reply by rob0101** [reply]
Anyone on Amber who has resorted to netzero (which shouldn't be needed for us) and cleared the high bar of getting an Amber api, probably isn't going to get too confused by the concept.
Those people are already committed to fixing issues with the Amber app.
